src/pager/index.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component, PropTypes } from 'react';
+import Icon from '../../infl-components/icon.jsx';
+import classnames from 'classnames';
+
+const MOVING_WINDOW_SIZE = 5;
+const EDGE_WINDOW_SIZE = 2;
+
+export default class Pager extends Component {
+
+  static propTypes = {
+    /**
+     * The current page number (starting at 1).
+     */
+    currentPage: PropTypes.number.isRequired,
```

Comment:
i think these propTypes are mostly self documenting.

a custom proptype for currentPage would be nice. warning if the current page does not within the range of number of pages derived from ( totalItemCount / perPage ) + 1